### ISSUE TITLE
Bundle TW.net into a deployable package

### ISSUE THREAD

**Issue by mplewis** [teslaworksumn/teslaworks.net#61]
uWSGI and nginx kinda suck to set up. Let's put together a [Docker container](http://www.docker.io/) that lets us start the server with one command.

**Comment by taylortrimble** [comment]
:+1:

**Comment by taylortrimble** [comment]
This will be interesting to tackle after #62 too, because I guess we will want that to auto-compile the SASS on/before deploy.

**Comment by taylortrimble** [comment]
Also, git commit hook goes here? Or another issue?
